hivemq-edge/src/main/java/com/hivemq/http/JaxrsHttpServer.java

```diff
@@ -91,6 +93,10 @@ public void startServer() {
                     System.setProperty(MAX_RESP_TIME, "45");
 
                     ResourceConfig resources = resourceConfig == null ? new ResourceConfig() : resourceConfig;
+                    // https://github.com/eclipse-ee4j/jersey/issues/2986
+                    // This server property tells jersey not to resolve relative location
+                    // so that / -> /app/ doesn't get resolved to http://domain/app/.
+                    resources.property(LOCATION_RELATIVE_RESOLUTION_DISABLED, true);
```

Comment:
with this set to true, the response headers will not look like `http://host:port/path/` but rather like `/path/` . This is less information, is this ok?

Reply:
Right. I'll adjust the wording.